source/_integrations/homekit.markdown

```diff
@@ -489,6 +542,14 @@ Media Player entities with `device_class: tv` will show up as Television accesso
 
 The volume and play/pause controls will show up on the Remote app or Control Center. If your TV supports volume control through Home Assistant, you will be able to control the volume using the side volume buttons on the device while having the remote selected on screen.
 
+#### Camera video is not streaming
+
+Ensure you have a recent version of FFmpeg installed. Verify that your stream is directly playable with `ffplay <stream_source>`. If you have changed your camera's entity configuration, you may need to [reset the accessory](#resetting-accessories).
```

Comment:
How would a Home Assistant user or a Home Assistant Core on Docker user do this?

Reply:
@xdissent Maybe suggest they use vlc to test the stream?

Reply:
I made the adjustment